src/west/manifest.py

```diff
@@ -67,6 +68,64 @@
 # Internal helpers
 #
 
+# Mapping of the architectures from platofrm.machine() to a common standard
```

Comment:
Do you mean, to some "Zephyr" standard? I agree "Darwin" is a little bit cryptic but `aaarch64` is not.